job_runner/reporting/file.py

```diff
@@ -198,4 +198,4 @@ def _dump_ok_file(self):
 
     def _dump_status_json(self):
         with open(self.STATUS_json, "w") as fp:
-            json.dump(self.status_dict, fp, indent=1)
+            json.dump(self.status_dict, fp, indent=4)
```

Comment:
There's a home made JSON parser somewhere. Just make sure that this file either is not read by that parser, or if it does, that the parser can deal with this indentation. 😅

Reply:
I was hoping for the test to detect that 😁

Reply:
🙏🏻

Reply:
I'm not even sure the C-layer reads this file anymore... 🤔 It definitively checks whether it exists (and removes it upon starting a forward model) to deduce running state, but I'm not sure it actually loads the file. It appears that this is all done in the Python layer... If so, there is no worries related to parsing it :)